website/addons/dryad/model.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from framework.exceptions import HTTPError
+
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+from website.addons.base import StorageAddonBase
+
+from website.addons.dryad import settings as dryad_settings
+#from website.addons.dryad import serializer as serializer
+
+from pyDryad import Dryad
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
+
+    def delete(self, **kwargs):
```

Comment:
The documentation states that this method needs to be instantiated. I haven't tried deleting it, but if you think it's OK, then I will